modules/migration-mtc-cr-manifests.adoc

```diff
@@ -268,6 +268,24 @@ spec:
 
 The `MigPlan` CR defines the parameters of a migration plan. It contains a group of virtual machines that are being migrated with the same parameters.
 
+[IMPORTANT]
```

Comment:
These two admonishments seem overzealous, but my first suggestion--adding this information in a single section of a concept module--and then linking to it faces the inconceivable challenge that we can't add a link from a module.

These admonishments seem to be the first instance of mentioning UIDs and GIDs.

Clearly, I'm not familiar with the material, but could this consideration about duplicate namespaces go on the pre-migration checklist page?  There is also some mention of namespaces on the "About the Migration..." page.  Maybe it could go there and is that page a place to mention what UID and GID ranges are and why they matter?

Reply:
These admonitions are not over-zealous. If the user does not know that UIDs/GIDs are migrated, they do not know why certain namespace combinations will fail validation. Also, it is only relevant for CLI, so there is not much point in putting the admonitions in another module.
No, this does not belong on premigration checklist because changing the target namespace is an edge case, not a normal migration ujse case.

A sys admin will know why duplicating UIDs/GIDs in a namespace is not a good idea. Can't be done. Migration will fail.